atomic_reactor/plugins/pre_resolve_composes.py

```diff
@@ -131,6 +133,36 @@ def adjust_for_autorebuild(self):
             self.log.info('Autorebuild detected: Ignoring compose_ids plugin parameter')
             self.compose_ids = tuple()
 
+    def get_arches(self):
+        # This will be replaced by a plugin soon but this will do for now
+        for plugin in self.workflow.buildstep_plugins_conf:
+            if plugin['name'] == PLUGIN_BUILD_ORCHESTRATE_KEY:
+                return plugin.get('args').get('platforms', [])
+
+    def validate_container_sets(self, data, container_sets, arches):
+        fail_msg = 'failed container set validation: '
+        data_rpms = []
+        for data_type in ['pulp_repos', 'packages', 'modules']:
```

Comment:
I don't understand what this validation method is trying to accomplish. Can you help me out?

Reply:
validate the container sets
confirm at all arches we intend to compose are in the container sets
confirm that for all RPMs we intend to compose, they are also in the container sets

if the purpose of container sets file is not stop composition when there are RPMs in the compose that aren't in the container sets, then I misread the docs again and would like an explanation of what's going on.

Reply:
Fixed by deletion.